homeassistant/components/recorder/statistics.py

```diff
@@ -1924,7 +1924,7 @@ def get_latest_short_term_statistics(
                 for metadata_id in missing_metadata_ids
                 if (
                     latest_id := cache_latest_short_term_statistic_id_for_metadata_id(
-                        run_cache, session, metadata_id
+                        run_cache, session, metadata_id, orm_rows=False
```

Comment:
False is fine here since we are in a read_only session

Reply:
Can you migrate all these PR comments to code comments? Future us will thank you.

Reply:
Will do 👍

Reply:
b49c572c50204f6f7927efe3252a5c6da62eb7f2